.jshintrc

```diff
@@ -1,6 +1,6 @@
 {
   // Set syntax to ES5 (e.g., complain about use of ES6 features)
-  "esversion": 6,
+  "esversion": 8,
```

Comment:
Can you update this comment to explain why ES8 is necessary?

Reply:
I got this error from JSHint: 
```bga_database/static/js/helloworld.js: line 2, col 34, 'async functions' is only available in ES8 (use 'esversion: 8').```

That's my mistake; I thought that async/await was part of ES6.